common/exception/ErrorMessage.java

```diff
@@ -194,8 +201,7 @@
     INVALID_VARIABLE_PREDICATE_STATE("Invalid state in variable predicate [%s] with answer [%s]: either a concept is missing or not an attribute."),
     NO_ATOMS_SELECTED("No atoms were selected from the query [%s]."),
     INVALID_CACHE_ENTRY("Query cache entry for query [%s] contains an invalid entry: [%s]."),
-    UNIFICATION_ATOM_INCOMPATIBILITY("Attempted unification on incompatible atoms."),
-    NON_EXISTENT_UNIFIER("Could not proceed with thr unification as the unifier doesn't exist."),
+    NON_EXISTENT_UNIFIER("Could not proceed with the unification as the unifier doesn't exist."),
```

Comment:
do we use it in the end :D?

Reply:
this is just a rename, got rid of the other one!